ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialLiveShareLayout.kt

```diff
@@ -0,0 +1,159 @@
+                Spacer(modifier = Modifier.minLinesHeight(5, MaterialTheme.typography.bodyMedium))
+            }
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+
+                Text(
+                    text = stringResource(title),
+                    style = MaterialTheme.typography.titleLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp),
+                    color = MaterialTheme.colorScheme.primary
+                )
+
+                Text(
+                    stringResource(body),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+            }
+        }
+        /*
+        Spacer for first two elements only
+        Animation will not be in box, will be element of parent column
+
+         */
+        if (page == Page.LIVE_SHARE_DESCRIPTION) {
+            if (img != null) {
+                Image(
+                    painter = painterResource(img), contentDescription = "",
+                    modifier = Modifier
+                        .height(dimensionResource(R.dimen.tutorial_page_live_share_anim_height))
+                        .fillMaxWidth()
+
+                )
+            }
+        } else {
+            if (anim != null) {
+                val composition by rememberLottieComposition(
+
+                    LottieCompositionSpec.RawRes(anim)
+                )
+
+                val progress by animateLottieCompositionAsState(
+                    // pass the composition created above
+                    composition,
+
+                    // Iterates Forever
+                    iterations = LottieConstants.IterateForever,
+
+                    // pass isPlaying we created above,
+                    // changing isPlaying will recompose
+                    // Lottie and pause/play
+                    isPlaying = true,
+
+                    // pass speed we created above,
+                    // changing speed will increase Lottie
+                    speed = 1f,
+
+                    // this makes animation to restart
+                    // when paused and play
+                    // pass false to continue the animation
+                    // at which is was paused
+                    restartOnPlay = false
+
+                )
+                LottieAnimation(
+                    composition,
+                    { progress },
+                    modifier = Modifier
+                        .height(dimensionResource(R.dimen.tutorial_page_live_share_anim_height))
+                        .fillMaxWidth()
+                )
+            }
+        }
+
+        Spacer(modifier = Modifier.weight(1f).fillMaxWidth())
```

Comment:
`.fillMaxWidth()` doesn't matter for the `Spacer` because in this use case we only care about the height